Kros.ProjectTemplates/src/Kros.CqrsTemplate/content/Startup.cs

```diff
@@ -47,6 +50,9 @@ public override void ConfigureServices(IServiceCollection services)
                 .AddClasses()
                 .AsMatchingInterface());
 
+            services.AddMvc(o => o.EnableEndpointRouting = false);
```

Comment:
In generally we don't want use Mvc. We add Mvc only in one service becaouse, actual Controllers routing doesn't support OData. 

So, please remove this line.

Reply:
Removed MVC.